src/output.rs

```diff
@@ -17,18 +19,23 @@ pub enum OutputType {
 
 impl OutputType {
     #[cfg(feature = "paging")]
-    pub fn from_mode(mode: PagingMode, pager: Option<&str>) -> Result<Self> {
+    pub fn from_mode(paging_mode: PagingMode, wrapping_mode: WrappingMode, pager: Option<&str>) -> Result<Self> {
         use self::PagingMode::*;
-        Ok(match mode {
-            Always => OutputType::try_pager(false, pager)?,
-            QuitIfOneScreen => OutputType::try_pager(true, pager)?,
+        use self::WrappingMode::*;
+        Ok(match paging_mode {
+            Always => OutputType::try_pager(false, wrapping_mode == Character, pager)?,
+            QuitIfOneScreen => OutputType::try_pager(true, wrapping_mode == Character, pager)?,
             _ => OutputType::stdout(),
         })
     }
 
     /// Try to launch the pager. Fall back to stdout in case of errors.
     #[cfg(feature = "paging")]
-    fn try_pager(quit_if_one_screen: bool, pager_from_config: Option<&str>) -> Result<Self> {
+    fn try_pager(
+        quit_if_one_screen: bool,
+        line_wrap: bool,
```

Comment:
As we don't have named parameters in Rust, I really like to avoid `bool` parameters. One is bad enough already (*"What does `OutputType::try_pager(false, pager)` do?"*), but two are definitely not ideal (I know that one was here already). It would be great if we could introduce two `enum`s instead. For example:
``` rust
enum ActionForShortFiles {
  QuitPager,
  Nothing,
}
```
or maybe just use the `less` wording with something like
``` rust
enum ChopLongLines {
  Yes,
  No,
}
```

This is a small point overall, but I think it would be great to improve this.

Reply:
I agree, I'll do it right away.